### PR TITLE
Added software tab for Linux installer

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - Installer doesn't have information about compiler

### CODE REVIEW

File: docs/install_guides/installing-openvino-linux.md [hunk 1]
```diff
@@ -33,6 +33,12 @@
   
   .. _Product Specifications: https://ark.intel.com/
 
+.. tab:: Software
+
+  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+  * GCC* 7.5.0 (for Ubuntu* 18.04) or GCC* 9.3.0 (for Ubuntu* 20.04)
+  * `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
+
```
Comment (ilya-lavrenov) [inline docs/install_guides/installing-openvino-linux.md:41]: @slyubimt do we need such info? Looks like it's not mandatory to have cmake.. not sure how to position others.

**Review by ilya-lavrenov — APPROVED** [review]

File: docs/install_guides/installing-openvino-linux.md (hunk 1 above)
Comment (slyubimt) [inline docs/install_guides/installing-openvino-linux.md:41]: We have Software tab for Windows install guide. I'm not sure how it's strict regarding GCC version, but in general it makes sense to add these requirements in order to align with software requirements defined in  the [Release notes](https://www.intel.com/content/www/us/en/developer/articles/release-notes/openvino-relnotes.html)

@ilyachur Can you add also this tab for APT (Ubuntu 18 and 20) and YUM (RHEL8) install guides?

Comment (nosovmik) [reply docs/install_guides/installing-openvino-linux.md:41]: @slyubimt @ilyachur 
Can we also add this to releases/2021/4 ? This is LTS release and users hit into such compiler compatibility issue: https://github.com/openvinotoolkit/openvino/issues/10929

Comment (ilyachur) [reply docs/install_guides/installing-openvino-linux.md:41]: Fixed

Comment (ilyachur) [reply docs/install_guides/installing-openvino-linux.md:41]: PR for 2021/4 https://github.com/openvinotoolkit/openvino/pull/11191

File: docs/install_guides/installing-openvino-apt.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ The complete list of supported hardware is available in the [Release Notes](http
 - Ubuntu 18.04 long-term support (LTS), 64-bit
 - Ubuntu 20.04 long-term support (LTS), 64-bit
 
+Software
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-apt.md:18]: ### 
```suggestion
**Software**
```

File: docs/install_guides/installing-openvino-apt.md [hunk 3]
```diff
@@ -15,6 +15,12 @@ The complete list of supported hardware is available in the [Release Notes](http
...
+
+- `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+- GCC* 7.5.0 (for Ubuntu* 18.04) or GCC* 9.3.0 (for Ubuntu* 20.04)
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-apt.md:21]:
```suggestion
- GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
```

File: docs/install_guides/installing-openvino-apt.md [hunk 4]
```diff
@@ -15,6 +15,12 @@ The complete list of supported hardware is available in the [Release Notes](http
 - Ubuntu 18.04 long-term support (LTS), 64-bit
 - Ubuntu 20.04 long-term support (LTS), 64-bit
 
+Software
+
+- `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-apt.md:20]: ### 
```suggestion
- [CMake 3.13 or higher, 64-bit](https://cmake.org/download/)
```

File: docs/install_guides/installing-openvino-apt.md [hunk 5]
```diff
@@ -15,6 +15,12 @@ The complete list of supported hardware is available in the [Release Notes](http
...
+- GCC* 7.5.0 (for Ubuntu* 18.04) or GCC* 9.3.0 (for Ubuntu* 20.04)
+- `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-apt.md:22]:
```suggestion
- [Python 3.6 - 3.9, 64-bit](https://www.python.org/downloads/windows/)
```

File: docs/install_guides/installing-openvino-linux.md [hunk 6]
```diff
@@ -33,6 +33,12 @@
   
   .. _Product Specifications: https://ark.intel.com/
 
+.. tab:: Software
+
+  * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+  * GCC* 7.5.0 (for Ubuntu* 18.04) or GCC* 9.3.0 (for Ubuntu* 20.04)
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-linux.md:39]: ### 
```suggestion
  * GCC 7.5.0 (for Ubuntu 18.04) or GCC 9.3.0 (for Ubuntu 20.04)
```

File: docs/install_guides/installing-openvino-yum.md [hunk 7]
```diff
@@ -14,6 +14,12 @@ The complete list of supported hardware is available in the [Release Notes](http
 
 - Red Hat Enterprise Linux 8, 64-bit
 
+Software
+
+- `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+- GCC* 8.2.0
+- `Python 3.6 - 3.9, 64-bit <https://www.python.org/downloads/windows/>`_
+
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-yum.md:22]: Please apply the same changes here. This is not included in Sphinx directives so we the hyperlinks need use Markdown markup. Or we can also use Sphinx directives to include this in a tab. 
Asterisks are not necessary for our doc site.

Comment (ilyachur) [reply docs/install_guides/installing-openvino-yum.md:22]: Fixed

**Review by xu-yuan1 — APPROVED** [review]

**Pull request merged**